src/proc/color-formats-converter.cpp

```diff
@@ -95,10 +95,11 @@ namespace librealsense
 
                 if (FORMAT == RS2_FORMAT_Y8)
                 {
-                    // Align all Y components and output 16 pixels (16 bytes) at once
-                    __m128i y0 = _mm_shuffle_epi8(s0, _mm_setr_epi8(1, 3, 5, 7, 9, 11, 13, 15, 0, 2, 4, 6, 8, 10, 12, 14));
-                    __m128i y1 = _mm_shuffle_epi8(s1, _mm_setr_epi8(0, 2, 4, 6, 8, 10, 12, 14, 1, 3, 5, 7, 9, 11, 13, 15));
-                    _mm_storeu_si128(&dst[i], _mm_alignr_epi8(y0, y1, 8));
+                    const __m128i vmask = _mm_set1_epi16( 0x00ff );
```

Comment:
Can you explain the issue?
We had color Y8 working a few month ago

Reply:
03/08 color Y8 on ROS
![image](https://github.com/IntelRealSense/librealsense/assets/64067618/8acac6af-69b7-4554-8c4b-6a4a619b9536)

Reply:
Oh now sure if SSE was enabled

Reply:
It wasn't working for me. I can show you.
I checked the non-SSE mode, and it works fine. It's only this specific code that doesn't work.
And I cannot say I understand the code that was there before. I understand this code I wrote.

Reply:
@SamerKhshiboun can you check if SSE is enabled on your NUC where you captured this recording?
This was a client request so I want to make sure we don't ruin it somehow (even on ROS)

Reply:
![image](https://github.com/IntelRealSense/librealsense/assets/54848706/af742df6-983d-4d13-b78d-c406aedd5494)

Reply:
I verified and found it work on D500 because we use M420 to Y8.
Issue is in YUYV to Y8